docusaurus/docs-internals/editor-controls/data-controls/radiogroup.md

```diff
@@ -0,0 +1,221 @@
+|:----------:|:----------------------------------:|:--------:|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+|    `id`    |              `string`              |    -     | The identifier of the key where the radioGroup will save your data                                                                                                                                                                                                               |
+|   `type`   |              `string`              |    -     | Type should be `"radioGroup"` to use this control                                                                                                                                                                                                                                |
+|  `label`   |              `string`              |    -     | The label displayed on the left side of the control                                                                                                                                                                                                                              |
+| `position` |              `number`              |    -     | The position of the control in toolbar                                                                                                                                                                                                                                           |
+| `devices`  | `"all"`/`"desktop"`/`"responsive"` | `"all"`  | Define the devices where the control will be rendered. Options: `"all"`, `"desktop"`, `"responsive"`. `"all"` renders the control on all devices. `"desktop"` renders the control only on desktop devices. `"responsive"` renders the control on both tablet and mobile devices. |
+| `disabled` |              `boolean`             | `false`  | Configure the condition under which the control is disabled or enabled.                                                                                                                                                                                                          |
+| `choices`  |          `Array<Choice>`           |    -     | `Choice: { title?: string, icon: string, value: L.Literal }`<br/>`title?` - The title of the radio element, which appears on hover.<br/>`icon` - The icon associated with the radio element.<br/>`value` - The value returned by the radio group when this option is selected.   |
```

Comment:
1. `L.Literal` -> `string | number`
2. `title?` -> `title`

preserve this structure 
![image](https://github.com/EasyBrizy/Brizy-Local-Editor/assets/52622998/daa42bd2-00a8-4860-a2ad-03ddc5d8fd3a)